docs/pages/common_issues/bundle_size.mdx

```diff
@@ -33,7 +33,17 @@ eg: `--arch=arm64 --platform=linux --target=18 --libc=glibc`
 
 ##### pdfjs
 
-If you need to use pdfjs, you should install it with `npm i pdfjs-dist--no-optional` because the optional dep: `canvas` takes about 180MB.
+- If you need to use pdfjs, you should install it with `npm i pdfjs-dist--no-optional` because the optional dep: `canvas` takes about 180MB.
+
+- If the above dosn't work (or gives some compilation errors) you can try:
+
+```js
+  experimental: {
+    outputFileTracingExcludes: {
+      "*": ["node_modules/canvas"],
```

Comment:
@revmischa was having build issues w/ the latest nextjs causing infinite loop. 🤷

Reply:
I filed an issue https://github.com/vercel/next.js/issues/60301

Reply:
People are reporting that updating sharp to v0.33.1 fixes this
Is this version specifier something that is inside open-next?

Reply:
https://github.com/sst/open-next/blob/main/packages/open-next/src/build.ts#L370

Reply:
@abhimskywalker do you mind updating the sharp version to the latest `0.33.2` and see if that fixes the issues?
https://github.com/lovell/sharp/releases/tag/v0.33.2